CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ### Added
 
+- Up can now be specified via `Setup.swift` https://github.com/tuist/tuist/issues/203 by @marciniwanicki & @kwridan
```

Comment:
I'd prefix this one with `**Breaking**`

Reply:
I've added `**Breaking**`, but wonder if `**[Breaking]**` wouldn't look better? Just realised there are 2 'Added' sections, would you mind if I merge them?